source/js/components/join/join.jsx

```diff
@@ -5,32 +5,19 @@ import basketSignup from '../../basket-signup.js';
 export default class JoinUs extends React.Component {
   constructor(props) {
     super(props);
+    // Default props defined at end of file
```

Comment:
I couldn't actually get this to work any other way with ES6 classes. Is there something I was missing?